src/main/java/be/syntra/devshop/DevshopFront/models/dtos/PaymentDto.java

```diff
@@ -0,0 +1,19 @@
+package be.syntra.devshop.DevshopFront.models.dtos;
+
+import be.syntra.devshop.DevshopFront.models.PaymentOption;
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+public class PaymentDto {
+
+    private BigDecimal totalPrice;
+    private List<PaymentOption> paymentOptions = new ArrayList<>(List.of(PaymentOption.values()));
```

Comment:
new ArrayList<>(List.of(PaymentOption.values())) -> Arrays.asList(PaymentOption.values())
Instead: don't... A dto is a data transfer object. Don't set values here...
Also also: I can understand why you would set the paymentOptions as a field on the Dto. It wouldn't matter when instantiating the template, given that you would simply add the paymentDto as a model, instead of two seperate ones. But ask yourself this: does my dto need to know about ALL the paymentOptions? If it is to determine the way of payment (for future reference or so...), wouldn't a field with a single value be enough? SoC-principle

Reply:
removed class here because next PR will put it in here